lib/static_sync/processor.rb

```diff
@@ -46,6 +46,8 @@ def handle_conflict(file)
         if file.cached?
           raise ConflictError, "modifications to existing cached files are not allowed."
```

Comment:
We are not using that api in the code at the moment. 

The assets we are building on production(this is true for the webpack assets at least for now) is based on the hash value of the content, if cache exists that actually means it's the same file, it's not override.

I'll work on the sprockets assets after the work for webpack is done.